### ISSUE TITLE
Placeholders does not replace in translation with not existing indexes

### ISSUE THREAD

**Issue by sszdh** [phalcon/cphalcon#10486]
Hi all,

I use `nativeArray`adapter for my translation. in my controller I try to translate a message and send it to user:

```php
$message = $this->translation->_("You are logged in as %username%", ['username' =>$user->username]);

```
If "`You are logged in as %username%`" key is not defined in language `array`, then the index returns as is, without replacement!

See this section of [source code](https://github.com/phalcon/cphalcon/blob/master/phalcon/translate/adapter/nativearray.zep#L65):

```php
	/**
	 * Returns the translation related to the given key
	 *
	 * @param string  index
	 * @param array   placeholders
	 * @return string
	 */
	public function query(string! index, placeholders = null) -> string
	{
		var translation;

		if fetch translation, this->_translate[index] {
			return this->replacePlaceholders(translation, placeholders);
		}
		return index;
	}
```

This issue does not append in `getext` addapter. You can see the [source](https://github.com/phalcon/cphalcon/blob/master/phalcon/translate/adapter/gettext.zep#L86). But in `csv`and `nativeArray` adapter already happens.

I think this is a Bug. 

Any idea?

Thanks

**Comment by andresgutierrez** [comment]
This is now fixed in the 2.0.3 branch

**Comment by sszdh** [comment]
@andresgutierrez  :+1:
